### ISSUE TITLE
kmlSubstyle balloonStyle display support for kmlFile layer placemarks

### ISSUE THREAD

**Issue by reterraform** [NASAWorldWind/WebWorldWind#839]
If I'm correct, kmlSubstyle balloonStyle is currently only parsed but not displayable in 
WebWorldWind kmlFile layer placemarks. If so, I'd like to suggest the capability to display them as a potential enhancement.

Thank you

**Comment by reterraform** [comment]
is this project still active? 

Thanks

**Comment by markpet49** [comment]
@reterraform Hi, Thanks for your interest in WorldWind. Yes, this project is under active development. We are currently working on a new set of enhancements and bug fixes that should show up in this repo in the next month or so. The enhancement you've requested will be reviewed for inclusion in a future release.

**Comment by balhar-jakub** [comment]
@reterraform You are right, at the moment the KmlBalloonStyle is unprocessed, but it should be relatively straightforward to implement it. There are needed tools within the Web World Wind. 

I can probably help you if you are interested in implementing the logic?

**Comment by reterraform** [comment]
Yes, any assistance with this would very much appreciated! I feel that, if implemented, it's a feature that will add a  lot of value to WebWorldWind.

CHEERS

**Comment by reterraform** [comment]
hello, just checking back to see if there's still any interest in implementing the KmlBalloonStyle functionality;

if not, no worries at all, & thanks in any case!

**Comment by balhar-jakub** [comment]
I was actually looking into it. It seems that this functionality is linked to the controls of the KML. What I can do, is update the KmlTreeControls and when a user clicks on the specific item in the tree if there is a KmlBalloonStyle associated with the feature, it will be displayed on the screen. 

Does this works well enough for your use case?

**Comment by reterraform** [comment]
that definitely sounds like it would indeed do the job; definitely greatly appreciate your kind time & effort on this matter!

CHEERS

**Comment by reterraform** [comment]
Hello again, can you direct me to where in the code that kmltree control can be modified? if so, I thought I'd try modifying my copy & see If I can get it to work there..

Thanks again and CHEERS in any case!

**Comment by balhar-jakub** [comment]
I am really sorry, I didn't get to the actual solution. The pandemic lockdowns limits my effectivity more than I thought.

This is the exact file: https://github.com/NASAWorldWind/WebWorldWind/blob/develop/src/formats/kml/controls/KmlTreeVisibility.js
